src/index.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+import Login from '.../Login';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
   <React.StrictMode>
+      <Login />
```

Comment:
We don't want this added here, we want this declared as another value in RoutesEnum (probably called RoutesEnum.LOGIN), and added into the MemoryRouter as both a route (like the others), and eventually as the default *: 
<img width="592" alt="image" src="https://user-images.githubusercontent.com/126940746/224431379-329ad052-0138-43eb-9128-be22b371b4a9.png">